tests/test_build_stub.nf.test.snap

```diff
@@ -0,0 +1,103 @@
+                "references/arriba/cytobands_hg38_GRCh38_v2.4.0.tsv",
+                "references/arriba/known_fusions_hg38_GRCh38_v2.4.0.tsv.gz",
+                "references/arriba/protein_domains_hg38_GRCh38_v2.4.0.gff3",
+                "references/ensembl",
+                "references/ensembl/Homo_sapiens.GRCh38.102.chr.gtf",
+                "references/ensembl/Homo_sapiens.GRCh38.102.dna.primary_assembly.fa",
+                "references/ensembl/Homo_sapiens.GRCh38.102.dna.primary_assembly.fa.fai",
+                "references/ensembl/Homo_sapiens.GRCh38.102.genepred",
+                "references/ensembl/Homo_sapiens.GRCh38.102.gtf",
+                "references/ensembl/Homo_sapiens.GRCh38.102.refflat",
+                "references/ensembl/Homo_sapiens.GRCh38.102_rrna_intervals.gtf.interval_list",
+                "references/fusion_report_db",
+                "references/fusion_report_db/cosmic.db",
+                "references/fusion_report_db/fusion_report.log",
+                "references/fusion_report_db/fusiongdb2.db",
+                "references/fusion_report_db/mitelman.db",
+                "references/fusioncatcher",
+                "references/fusioncatcher/human_v102",
+                "references/hgnc",
+                "references/hgnc/HGNC-DB-timestamp.txt",
+                "references/hgnc/hgnc_complete_set.txt",
+                "references/star",
+                "references/star/Genome",
+                "references/star/Log.out",
+                "references/star/SA",
+                "references/star/SAindex",
+                "references/star/chrLength.txt",
+                "references/star/chrName.txt",
+                "references/star/chrNameLength.txt",
+                "references/star/chrStart.txt",
+                "references/star/exonGeTrInfo.tab",
+                "references/star/exonInfo.tab",
+                "references/star/geneInfo.tab",
+                "references/star/genomeParameters.txt",
+                "references/star/sjdbInfo.txt",
+                "references/star/sjdbList.fromGTF.out.tab",
+                "references/star/sjdbList.out.tab",
+                "references/star/transcriptInfo.tab",
+                "references/starfusion",
+                "references/starfusion/ctat_genome_lib_build_dir",
+                "references/starfusion/ref_annot.cdna.fa",
+                "rrnatranscripts",
+                "rrnatranscripts/Homo_sapiens.GRCh38.102_rrna_intervals.gtf"
+            ],
+            [
+                "Homo_sapiens.GRCh38.102_rrna_intervals.gtf.bed:md5,d41d8cd98f00b204e9800998ecf8427e",
+                "Homo_sapiens.GRCh38.102.dna.primary_assembly.dict:md5,d41d8cd98f00b204e9800998ecf8427e",
+                "Homo_sapiens.GRCh38.102.gff3:md5,d41d8cd98f00b204e9800998ecf8427e",
+                "cytobands_hg38_GRCh38_v2.4.0.tsv:md5,d41d8cd98f00b204e9800998ecf8427e",
+                "protein_domains_hg38_GRCh38_v2.4.0.gff3:md5,d41d8cd98f00b204e9800998ecf8427e",
```

Comment:
I don't know if it makes sense to snapshot md5sum as they are all the same because they are empty...

Reply:
I see what you mean and how this could get confusing after we update the test profiles. I'll remove it